### DISCUSSION TITLE
Support an AutoHotKey like syntax for matches

### DISCUSSION THREAD

**Discussion by federico-terzi** [espanso/espanso#1183] [Feature requests and Ideas]
As suggested in this [reddit post](https://www.reddit.com/r/espanso/comments/fxa5n6/please_federico_adopt_a_syntax_similar_to/) a (partial) support of the AutoHotKey syntax could make matches simpler/faster to add

**Comment by Xander314** [comment]
I would certainly be interested in this feature as well!

**Comment by qalisto** [comment]
Lissy93 created a [promising tool](https://github.com/Lissy93/ahk-to-espanso) for conversion.

Federico, please take note as I believe many people would come to Espanso from AHK for cross-platform reasons.  ##B
